_includes/master/kubernetes/k8s_binary_init.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
That script copies the downloaded binaries into their final location and makes them executable.  This is a different responsibility than retrieving the binaries.  I think I see where you are going but I don't know how the same script would be used both for fetching the binaries in a Vagrant VM and could be used externally (on the host running the VMs), at least without putting in some kind of similar logic.